Lombiq.Tests.UI/OrchardCoreUITestBase.cs

```diff
@@ -261,6 +262,10 @@ public abstract class OrchardCoreUITestBase
             BrowserConfiguration = { Browser = browser },
         };
 
+        ProgramCli.DefaultShellCliCommandFactory = OSDependentShellCliCommandFactory
+            .UseCmdForWindows()
+            .UseForOtherOS(new BashShellCliCommandFactory("-login"));
```

Comment:
Does this need to be configured statically? If yes, why not in the static ctor? Also, it would be better not here but in `AtataFactory`.

Reply:
Yes, it's a static config. I moved it to the static ctor.